web/src/cypress/integration/favorites.ts

```diff
@@ -0,0 +1,107 @@
+import { testScreen } from '../support'
+import { Chance } from 'chance'
+const c = new Chance()
+
+testScreen('Favorites', testFavorites)
+
+function testFavorites(screen: ScreenFormat) {
```

Comment:
If we create a new test integration file just for favorites, we will want to move the service favorites tests over here as well, and since the tests between these will be mostly the same, we can have a function that accepts a couple params that handles doing everything, then within each `describe` we would have something like

(pseudo code):

```
describe Rotation Favorites {
  it should have favored rotations move to first on rotation list:
    testSortOrder(rot.name)

  it should be able to set as a favorite
    testSettingFavorite(rot.id)

  // other rotation specific tests
}

describe Service Favorites {
  . . .
}
```